data/apply-planet_disputed_area_features_name_suppression.sql

```diff
@@ -0,0 +1,84 @@
+-- Script to delete names from certain disputed locations after osm2pgsql import.
+-- The "bbox" query creates polygon geometries to define which areas to select for name removal
+-- The "suppress" query selects features from the planet table that are fully within a "bbox" polygon
+-- The "update" query then deletes any key value pair within the selection with "name" somewhere in the key's string.
+
+with bbox as
+    (select *
+    from (values
+        (st_transform(st_geomfromtext('POLYGON ((-78.89 15.59,-78.89 16.12,-78.30 16.12,-78.30 15.59,-78.89 15.59))', 4326), 3857)),
```

Comment:
Is there a way to comment which disputed areas each polygon is for? Inline in the SQL or at least in a comment above the SQL?

Reply:
I assume these are almost all island disputes, given their bounding boxes. But would you support this method for larger arbitrary shapes for a land dispute, too?

Reply:
Yes, these are currently all island or atoll disputes, but I believe it would also work for a land area just as well. 

The selection only takes features fully within the bounding box so I believe larger features like relations should not be unexpected casualties.

Reply:
I updated the query with labels for each polygon